### PR TITLE
added logo and text 

### PR DESCRIPTION
**Pull request by aselviar**
resized logo

resized logo

added text in readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,13 +4,17 @@
...
+    <br />
+    <img src='ekz_logo.png' width="220">
 </p>
 
 ---
 
 The `smartmeter-datacollector` tool is a Python3 software which allows you to continuously retrieve data from supported smart meters (listed below).
 The acquired values can be forwarded to one or more data sinks like a MQTT broker or logger.
 
+This tool is an open source software funded by **[EKZ (Elektrizitätswerke des Kantons Zürich)](https://www.ekz.ch)** and developed by **[Supercomputing Systems AG](https://www.scs.ch)**. The goal of this voluntary initiative is to create a tool that can read, decode and decrypt data (if necessary) from the local interface of various smart meters and convert it into human-readable and simple formats. This should eliminate the complexity of decoding DLMS and similar protocols for the user. The user has the data in a format, which he can easily use in various use-cases ranging from simple vizualization (see [Method 1: Raspberry Pi image with demo](#method-1-raspberry-pi-image-with-demo)) to integration in energy management algorithms or in custom home automation software.
```
Comment (Montellese) [inline README.md:16]: `[...] to integration into energy management algorithms or into custom home automation software.`

File: README.md [hunk 2]
```diff
@@ -4,13 +4,17 @@
     <a href="LICENSE"><img alt="License: GPL-2.0-only" src="https://img.shields.io/badge/license-GPLv2-blue.svg"></a> <a href="https://github.com/scs/smartmeter-datacollector/pulls"><img alt="Pull Requests Welcome" src="https://img.shields.io/badge/PRs-welcome-brightgreen.svg"></a> <a href="https://github.com/scs/smartmeter-datacollector/pulls"><img alt="Contributions Welcome" src="https://img.shields.io/badge/contributions-welcome-brightgreen.svg"></a>
     <br />
     <img alt="Python Code Checks" src="https://github.com/scs/smartmeter-datacollector/actions/workflows/python-code-checks.yml/badge.svg?branch=master"> <a href="https://pypi.org/project/smartmeter-datacollector/"><img alt="PyPI - Python Version" src="https://img.shields.io/pypi/pyversions/smartmeter-datacollector"></a>
+    <br />
```
Comment (Montellese) [inline README.md:7]: Can you remove the white border around the blue EKZ logo? When using Github with the dark theme it looks quite ugly due to the white border ;-)

**Review by Montellese — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (aselviar) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (aselviar) [reply README.md:7]: done

Comment (aselviar) [reply README.md:7]: can you please resize the icon in the readme - make it a bit smaller? For some reason I cannot fix it

Comment (Montellese) [reply README.md:7]: To me it looks like `width="200"` is working properly. Do you want it even smaller? If so you have to use a smaller number (e.g. 100).

Comment (Montellese) [reply README.md:7]: Is the size OK for you now or does it have to be smaller?

Comment (aselviar) [reply README.md:7]: I would put width=120

Comment (Montellese) [reply README.md:7]: OK then you'll have to make the change and push a new commit. I can squash all commits together for the merge so don't worry about that.

Comment (aselviar) [reply README.md:7]: done, thanks!

**Review by Montellese — APPROVED** [review]
Looks good now, thanks.

**Pull request merged**